### DISCUSSION TITLE
Bug ? Summary of properties of associations between classes are not returned by py-capellambse

### DISCUSSION THREAD

**Discussion by khammoutene** [dbinfrago/py-capellambse#191] [Q&A] [answered]
Is it a bug?

I have put some text in the summary field of an association in a class diagram but I cannot find it in the .summary attribute of a property belonging to a class while I see that the name of the association is correctly returned.

Can somebody check?

Thank you

**Comment by ewuerger** [comment, marked as answer]
Yes, this will be solved in one go with PR #194. I advise you to have a precise look at the notebook that is referred there. You can see the effects of the changes there that will come live when it is reviewed and merged into master.

**Reply by khammoutene** [reply]
Thank you. I will

**Comment by khammoutene** [comment]
I have updated to 0.5.4.dev8 but I still don't get the value of the summary attribute for Property which represents Associations while I get them from summary attributes representing Class attributes in a class diagram.

pip install capellambse==0.5.4.dev8
Requirement already satisfied: capellambse==0.5.4.dev8 in c:\apps\py-capellambse-master\.venv\lib\site-packages (0.5.4.dev8)
Requirement already satisfied: lxml>=4.5.0 in c:\apps\py-capellambse-master\.venv\lib\site-packages (from capellambse==0.5.4.dev8) (4.9.1)
Requirement already satisfied: typing-extensions<5,>=4.0.0 in c:\apps\py-capellambse-master\.venv\lib\site-packages (from capellambse==0.5.4.dev8) (4.4.0)
Requirement already satisfied: Pillow>=7.0.0 in c:\apps\py-capellambse-master\.venv\lib\site-packages (from capellambse==0.5.4.dev8) (9.2.0)
Requirement already satisfied: markupsafe>=1.1 in c:\apps\py-capellambse-master\.venv\lib\site-packages (from capellambse==0.5.4.dev8) (2.1.1)
Requirement already satisfied: platformdirs>=1.4.1 in c:\apps\py-capellambse-master\.venv\lib\site-packages (from capellambse==0.5.4.dev8) (2.5.2)
Requirement already satisfied: svgwrite>=1.3.1 in c:\apps\py-capellambse-master\.venv\lib\site-packages (from capellambse==0.5.4.dev8) (1.4.3)

**Reply by ewuerger** [reply]
Hi,

the referenced issue is still open and the PR not merged in since a review of another code owner is needed. @Wuestengecko, our development lead is currently on vacation and will be back after the 11th. What you can do is the following to install capellambse with the changes from the PR:
```
pip install git+https://github.com/DSD-DBS/py-capellambse.git@pull/194
```

Or you just wait till the issues are closed.

**Comment by khammoutene** [comment]
Thank you.

I have tried to install the changes but got these errors:

![image](https://user-images.githubusercontent.com/27721103/201298797-b3d39a87-cd4a-4ed4-903f-84ed2ae92059.png)

**Reply by ewuerger** [reply]
Woopsie. My bad. Try:
```
pip install git+https://github.com/DSD-DBS/py-capellambse.git@add-property-source-from-association
```
instead. I thought `@pull/194` would already work but pip doesn't understand this specific GitHub reference yet.
